internal/report/source.go

```diff
@@ -595,18 +602,31 @@ func openSourceFile(path, searchPath string) (*os.File, error) {
 }
 
 // trimPath cleans up a path by removing prefixes that are commonly
-// found on profiles.
-func trimPath(path string) string {
-	basePaths := []string{
-		"/proc/self/cwd/./",
-		"/proc/self/cwd/",
+// found on profiles plus configured prefixes.
+func trimPath(path, trimPath, searchPath string) string {
+	sPath, searchPath := filepath.ToSlash(path), filepath.ToSlash(searchPath)
```

Comment:
+1 to changing path.

-1 to exposing the slash change to the API. I think it simpler to leave all the slash-specific code inside the implementation of this function, unless there is a good reason.

Reply:
Yes, like @rauls5382 says, I think I am more comfortable if the slash machinery stays fairly isolated within this function. I am not super excited about introducing a precondition that the input arguments are slashized by the caller.

For a similar reason we keep the original value of `path` here and keep `sPath` separate. The former is used for the return value so that slashes are the same as the caller gave. I added a comment to make it clearer.